app/utility/base_planning_svc.py

```diff
@@ -97,8 +119,7 @@ async def remove_links_missing_facts(links):
         :param links:
         :return: updated list of links
         """
-        links[:] = [l for l in links if
-                    not re.findall(r'(#{[a-zA-Z1-9]+?\..+?})', b64decode(l.command).decode('utf-8'), flags=re.DOTALL)]
+        links[:] = [l for l in links if not BasePlanningService.re_variable.findall(b64decode(l.command).decode('utf-8'))]
```

Comment:
The `\.` in the original regex was causing the function to not identify facts that didn't have periods in them.  This caused them to not be filtered out correctly, leading to a bunch of errors when running an operation. Since we no longer require A.B.C formatting, the pattern needed to recognize facts like `#{foobar}`.